### DISCUSSION TITLE
NocoDB database backup [Jun 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2432] [Discord Threads]
> 🙋‍♂️ **Ra**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Ra**  
#### <@857998597474484295> I’m currently using Heroku but might move to a Digital Ocean Droplet later. Would be happy to know the options available  
#  
> 💁 **o1lab**  
#### You would have do that manually. Not advisable. Cloud providers provide backups. Refer to their sites  
#  
> 🙋‍♂️ **peter piper**  
#### if hosted locally, a cron tab to export to a sql file  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984993367496749116)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
